codalab/worker_manager/kubernetes_worker_manager.py

```diff
@@ -74,6 +74,12 @@ def __init__(self, args):
         configuration.api_key['authorization'] = args.auth_token
         configuration.host = args.cluster_host
         configuration.ssl_ca_cert = args.cert_path
+        if configuration.host == "https://codalab-control-plane:6443":
```

Comment:
Allows anonymous access to the cluster without SSL checks, for testing / development purposes only. I did this because it was getting really complicated to provision the right SSL certs -- we might in the future see if there's an automated / easy way to provision SSL certs for communication with the local cluster.

Reply:
Sounds good, should we add a comment?

Reply:
Yes, done